package.json

```diff
@@ -15,5 +15,8 @@
   "bugs": {
     "url": "https://github.com/FrahOrru/Orru-Francesca-HS/issues"
   },
-  "homepage": "https://github.com/FrahOrru/Orru-Francesca-HS#readme"
+  "homepage": "https://github.com/FrahOrru/Orru-Francesca-HS#readme",
+  "dependencies": {
+    "commander": "^10.0.1"
```

Comment:
`axios` is not in dependencies, also you could add `bin` (probably, in a separate package) to make it a standalone cli